Xamarin.Forms.Platform.Android/FastRenderers/LabelRenderer.cs

```diff
@@ -292,9 +292,9 @@ void UpdateFont()
 			}
 		}
 
-		void UpdateTextDecorations()
+		void UpdateTextDecorations(bool isNewElement)
 		{
-			if (!Element.IsSet(Label.TextDecorationsProperty))
+			if (!Element.IsSet(Label.TextDecorationsProperty) && isNewElement)
```

Comment:
I'm thinking that checking for IsSet isn't the most reliable way to check for the TextDecorationsProperty. 
Wouldn't checking *if Label.TextDecoration == TextDecorations.None*
Be more accurate?

Reply:
That would prevent you from explicitly removing the text decorations after they had been set for the first time.

Reply:
actually wait, no it wouldn't haha. I've locked myself into the line of thinking that I had before adding isNewElement. I can change this too.